File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/Inline_object.scala

package org.openapitools.client.model

import java.io.File

case class Inline_object (
    _model: Option[SkinModel],
    /* The skin image in PNG format */
    _file: File
)
object Inline_object {
    def toStringBody(var_model: Object, var_file: Object) =
        s"""
        | {
        | "model":$var_model,"file":$var_file
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/org/openapitools/model/InlineObject.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.SkinModel
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param model 
 * @param file The skin image in PNG format
 */
data class InlineObject (

        @get:NotNull 
        @JsonProperty("file") val file: java.io.File,

        @JsonProperty("model") val model: SkinModel? = null
) {

}

